ccd-definition/AuthorisationComplexType/caseDocuments/AdditionalApplicationsBundle.json

```diff
@@ -1403,6 +1403,34 @@
       }
     ]
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "additionalApplicationsBundle",
+    "ListElementCode": "c2DocumentBundle.requestedHearingToAdjourn",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[LABARRISTER]",
+          "[SOLICITOR]",
```

Comment:
I wonder if we need SOLICITORA to J.

Reply:
Ah yeah I think so, well caught! 

It's a weird file though, as I copied another fields permissions and none of them seemed to have SOLICITORA->J, wonder if there's any undiscovered issues because of that.